controllers/front/CartController.php

```diff
@@ -432,6 +432,35 @@ protected function processChangeProductInCart()
                 'Shop.Notifications.Error'
             );
         }
+        
+        // Check minimal_quantity
+        if(!$this->id_product_attribute){
+            if($qty_to_check < $product->minimal_quantity){
+                 array_push(
+                     $this->{$ErrorKey},
+                     $this->trans(
+                         'The minimum purchase order quantity for the product %product% is %quantity%.',
```

Comment:
Is it necessary to add the product name here? I mean, it is already mentioned on the product page... perhaps we can just keep the already translation key _The minimum purchase order quantity for the product is %quantity%._ in _Shop.Theme.Checkout_?

Reply:
hello, could you please keep the `$this->{$ErrorKey}[]` notation? not that I dislike `array_push` but for consistency with rest of the code in this method

Reply:
More array_push have poor performance when used with one element.

Reply:
Saw both being used in the previous version (the one I tested on, even before the file moved to Presenter folder if i'm not mistaken). But I agree, no need for array_push.